source/user-manual/indexer-api/use-case.rst

```diff
@@ -506,13 +506,13 @@ The query result below shows the value of ``hits.total.value`` key indicates tha
    }
 
 Threat hunting
-^^^^^^^^^^^^^^
+--------------
 
 The Wazuh indexer API is helpful during threat hunting exercises where you have to query external systems. This use case demonstrates how we can extract source IPs from alerts and run them against AbuseIPDB; we generate a report showing if the source IP is contained in AbuseIPDB’s bad reputation IP listing. Follow the steps below.
 
-1. Sign up for a free `AbuseIPDB account <https://www.abuseipdb.com/register?plan=free>`_ and obtain an `AbuseIPDB API key <https://www.abuseipdb.com/account/api>`_.
+#. `Sign up <https://www.abuseipdb.com/register?plan=free>`_ for a free AbuseIPDB account and obtain an `AbuseIPDB API <https://www.abuseipdb.com/account/api>`_ key.
```

Comment:
```suggestion
#. `Sign up <https://www.abuseipdb.com/register?plan=free>`__ for a free AbuseIPDB account and obtain an `AbuseIPDB API <https://www.abuseipdb.com/account/api>`__ key.
```

Reply:
Better use anonymous hyperlinks to link to external pages by using double underscores